sw/device/silicon_creator/lib/sigverify_tests/sigverify_set_testvectors.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+import hjson
+from Crypto.Hash import SHA256
+from mako.template import Template
+
+'''
+Read in a JSON test vector file, convert the test vector to C constants, and
+generate a header file with these test vectors.
+'''
+
+# Number of 32-bit words in a 3072-bit number
+RSA_3072_NUMWORDS = int(3072 / 32)
+
+# Number of 32-bit words in a 256-bit number
+INT_256_NUMWORDS = int(256 / 32)
+
+# Template file name
+TEMPLATE = 'sigverify_testvectors.h.tpl'
+
+# Default output file name
+DEFAULT_OUTFILE = 'sigverify_testvectors.h'
+
+
+def compute_n0_inv(n):
+    '''Compute -(n^-1) mod 2^256, a Montgomery constant.
+
+    Sigverify expects this constant to be precomputed. Uses the standard Dusse
+    and Kaliski method; see https://eprint.iacr.org/2017/1057.pdf for more
+    details.
+    '''
+    w = 256
+    y = 1
+    # n0 = n mod 2^256
+    n0 = n & ((1 << 256) - 1)
+    # maski = 2^i - 1 (start with i=2)
+    maski = 3
+    for i in range(2, w+1):
+        # (n * y) mod 2^i = (n0 * y) mod 2^i because i <= 256
+        if (n0 * y) & maski != 1:
+            y = y + (1 << (i-1))
+        maski <<= 1
+        maski += 1
+    return (1 << w) - y
```

Comment:
I know this is the standard method but I think `pow(-n, -1, 2**256)` should also work here. Any reason for not using that?